Example/PaymentSheet Example/PaymentSheet Example/EndpointSelectorViewController.swift

```diff
@@ -0,0 +1,187 @@
+
+    override func viewDidLoad() {
+        loadEndpointSelector(endpoint: endpointSelectorEndpoint)
+        setBarButtonItems()
+    }
+
+    func setBarButtonItems() {
+        let barButtonItem = UIBarButtonItem(title: "Set Manually", style: .plain, target: self, action: #selector(didTapSetManually))
+        navigationItem.rightBarButtonItems = [barButtonItem]
+        let cancel = UIBarButtonItem(barButtonSystemItem: .cancel, target: self, action: #selector(didTapCancel))
+        navigationItem.leftBarButtonItems = [cancel]
+    }
+
+    @objc func didTapCancel() {
+        self.selectorDelegate?.cancelTapped()
+    }
+
+    @objc func didTapSetManually() {
+        let alertController = UIAlertController(title: "Set Endpoint Manually", message: nil, preferredStyle: .alert)
+        alertController.addTextField { textField in
+            textField.text = self.currentCheckoutEndpoint
+        }
+
+        let submitAction = UIAlertAction(title: "Submit", style: .default) { [unowned alertController] _ in
+            guard let textFields = alertController.textFields,
+                  let input = textFields[0].text else {
+                return
+            }
+            self.selectorDelegate?.selected(endpoint: input)
+            self.navigationController?.popViewController(animated: true)
+        }
+        let cancelAction = UIAlertAction(title: "Cancel", style: .cancel)
+        alertController.addAction(submitAction)
+        alertController.addAction(cancelAction)
+        present(alertController, animated: true)
+    }
+}
+
+// MARK: - TableViewController
+extension EndpointSelectorViewController {
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        guard let specs = endpointSpecs else {
+            return 2
+        }
+        return specs.endpointMap.count + 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection sectionNumber: Int) -> Int {
+        if sectionNumber == 0 {
+            return 1
+        }
+        let sectionNumberIndex = sectionNumber - 1
+        guard let specs = endpointSpecs,
+              let sortedSection = sortedSections() else {
+            return 0
+        }
+        let sectionKey = sortedSection[sectionNumberIndex].key
+        return specs.endpointMap[sectionKey]?.count ?? 0
+    }
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection sectionNumber: Int) -> String? {
+        if sectionNumber == 0 {
+            return "Current Endpoint"
+        }
+
+        let sectionNumberIndex = sectionNumber - 1
+        guard let sortedSection = sortedSections() else {
+            return "Loading..."
+        }
+        return sortedSection[sectionNumberIndex].key
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let reuseIdentifier = "endpointCell"
+        let cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier) ??
```

Comment:
use the variant that takes the index path, that always returns a valid cell

Reply:
I tried using:
```
        let cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier, for: indexPath)
```
I'm seeing:
```
libc++abi: terminating with uncaught exception of type NSException
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'unable to dequeue a cell with identifier endpointCell - must register a nib or a class for the identifier or connect a prototype cell in a storyboard'
```

Reply:
![Screen Shot 2022-12-08 at 5 01 46 PM](https://user-images.githubusercontent.com/99628984/206599131-ed2a1822-2352-4127-9824-d3b290e1dd49.png)

Reply:
Ah, ok. Added
```
self.tableView.register(UITableViewCell.self, forCellReuseIdentifier: reuseIdentifierEndpointCell)
```
in `viewDidLoad`